From b2f10174c48333f7a0ce8c01b86536f959d92348 Mon Sep 17 00:00:00 2001
From: Lukas Bulwahn <lukas.bulwahn@oss.bmw-carit.de>
Date: Wed, 21 Sep 2016 13:16:07 +0200
Subject: [PATCH] address gcc6 build error

With gcc6, compiling fails with `stdlib.h: No such file or directory`,
as including '-isystem /usr/include' breaks with gcc6, cf.,
https://gcc.gnu.org/bugzilla/show_bug.cgi?id=70129.

This commit addresses this issue for this package in the same way
it was addressed in various other ROS packages. A list of related
commits and pull requests is at:

  https://github.com/ros/rosdistro/issues/12783

Signed-off-by: Lukas Bulwahn <lukas.bulwahn@oss.bmw-carit.de>

Upstream-Status: Accepted [https://github.com/ros/dynamic_reconfigure/commit/b2f10174c48333f7a0ce8c01b86536f959d92348]

This patch has been generated with:
  `git format-patch -1 b2f10174c48333f7a0ce8c01b86536f959d92348`
in the dynamic_reconfigure repository.

Signed-off-by: Lukas Bulwahn <lukas.bulwahn@oss.bmw-carit.de>
---
 CMakeLists.txt | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ce896b9..60c6333 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -4,8 +4,7 @@ project(dynamic_reconfigure)
 find_package(catkin REQUIRED COMPONENTS message_generation roscpp std_msgs)
 find_package(Boost REQUIRED COMPONENTS system thread)
 
-include_directories(include ${catkin_INCLUDE_DIRS})
-include_directories(SYSTEM ${Boost_INCLUDE_DIRS})
+include_directories(include ${catkin_INCLUDE_DIRS} ${Boost_INCLUDE_DIRS})
 
 catkin_python_setup()
 
-- 
2.5.5

